MA-Bylaws.md

```diff
@@ -185,14 +175,16 @@ the Assembly are required at two consecutive Assembly meetings each.
 A decision on dissolution may only be made after the matter has been
 approved by The Board.
 
+The remaining funds will be donated to other non-profit organizations with similar goals as MA.
```

Comment:
Should the decision makers be clarified? Board or Assembly?

Reply:
Too late, these bylaw changes have already been approved. 
Hopefully we will not have to dissolve MA soon, otherwise we will decide then ;-)

Reply:
This was also legal requirement, and I also hope that we don't have to test it.
However, it would seem like a board decision, unless the assembly gave a clear direction.